.github/workflows/swift-toolchain.yml

```diff
@@ -213,7 +228,64 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        arch: ['amd64', 'arm64', 'x86']
+        include:
+          - arch: amd64
+            cc: cl
+            cflags: ${{ needs.context.outputs.CMAKE_C_FLAGS }}
+            cxx: cl
+            cxxflags: ${{ needs.context.outputs.CMAKE_CXX_FLAGS }}
+            os: Windows
+            extra_flags: 
+
+          - arch: arm64
+            cc: cl
+            cflags: ${{ needs.context.outputs.CMAKE_C_FLAGS }}
+            cxx: cl
+            cxxflags: ${{ needs.context.outputs.CMAKE_CXX_FLAGS }}
+            os: Windows
+            extra_flags: 
+
+          - arch: x86
+            cc: cl
+            cflags: ${{ needs.context.outputs.CMAKE_C_FLAGS }}
+            cxx: cl
+            cxxflags: ${{ needs.context.outputs.CMAKE_CXX_FLAGS }}
+            os: Windows
+            extra_flags: 
+
+          - arch: arm64
+            cc: clang
+            cflags: -ffunction-sections -fdata-sections -g
```

Comment:
Derive from `CMAKE_ANDROID_C_FLAGS` / `CMAKE_ANDROID_CXX_FLAGS` to avoid repeating?